### PR TITLE
Fixing dijkstra ordering issue

### PR DESCRIPTION
**Pull request by krashish8**
dijkstra on #68.

Changes proposed in this pull request:
- Adds pgTAP tests for dijkstra directory that check the output after rows ordering.

@pgRouting/admins

### CODE REVIEW

**Comment by krashish8** [comment]
As already mentioned, the `dijkstraVia` function crashes the server when the rows are sorted (Issue #67)
So, currently, I have commented those row-ordering tests that crash the server.

File: pgtap/dijkstra/dijkstraVia-rows-check.sql [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+SELECT * FROM pgr_dijkstraVia(
+    'SELECT id, source, target, cost, reverse_cost
+    FROM edge_table
+    ORDER BY RANDOM()',
+    ARRAY[7, 3, 2, 12, 7], strict => false, U_turn_on_edge => false,
+    directed => false
+);
+
+-- Directed
+
```
Comment (cvvergara) [inline pgtap/dijkstra/dijkstraVia-rows-check.sql:234]: Wrap all the tests of dijkstraVia with a TODO
otherwise we will never see again the server crash.

**Comment by cvvergara** [comment]
DijkstraVia is proposed, so dont fix.

**Comment by cvvergara** [comment]
Or shall I say, dont try to fix

**Comment by cvvergara** [comment]
Only dijkstra/dijkstraCost is to be fixed

**Comment by krashish8** [comment]
Line Graph docqueries failed after sorting the Dijkstra rows. So, I fixed it.

**Pull request merged**